Review contributions (#6)
lsf37/website: Contribute/style.md

pre: Coding Conventions
pre_link: conventions.html
sub: '<p>

  Formatting, style, and C language subset requirements, divided into <a
  href="#general-c-code">general</a>, <a href="#user-level-code">user-level
  code</a>, and <a href="#kernel-code">kernel code</a> (including specific
  requirements for <a href="#verification-requirements">verified kernel

>   code</a>).

Author: It's for any verified code, not just kernel code

Author: Fixed that one up and added more navigation to the page in general. Also tweaked layout and parentheses a bit more throughout.

</p>'

---

We strive to follow the below conventions for all the code maintained by the
seL4 Foundation in the <a href="https://github.com/seL4">seL4 GitHub org</a>.

> We make an exception for external libraries, where we refer to the existing
> conventions.

Author: ```suggestion
We make an exception for external libraries, where we use their existing
conventions.
```

Author: I think I want to tweak this a bit more. Attempt and strive are too weak -- they are actual requirements, we just have not applied them to old code.
